docs/utilities/parameters.md

```diff
@@ -159,260 +175,240 @@ SSM Parameter Store | `SSMProvider.getMultiple(String)`
 Secrets Manager | `SecretsProvider.get(String)` `SecretsProvider.get(String, Class)`   | `secretsmanager:GetSecretValue`
 DynamoDB | `DynamoDBProvider.get(String)` `DynamoDBProvider.getMultiple(string)` | `dynamodb:GetItem` `dynamoDB:Query`
 
-## SSM Parameter Store
+## Retrieving Parameters
+You can retrieve parameters either using annotations or by using the `xParamProvider` class for each parameter
+provider directly. The latter is useful if you need to configure the underlying SDK client, for example to use
+a different region or credentials, the former is simpler to use.
 
-You can retrieve a single parameter using SSMProvider.get() and pass the key of the parameter.
-For multiple parameters, you can use SSMProvider.getMultiple() and pass the path to retrieve them all.
+### Using Annotations
 
-Alternatively, you can retrieve an instance of a provider and configure its underlying SDK client,
-in order to get data from other regions or use specific credentials.
+=== "Secrets Manager: @SecretsParam"
 
-=== "SSMProvider"
-
-    ```java hl_lines="6"
-    import software.amazon.lambda.powertools.parameters.SSMProvider;
-    import software.amazon.lambda.powertools.parameters.ParamManager;
-
-    public class AppWithSSM implements RequestHandler<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEvent> {
-        // Get an instance of the SSM Provider
-        SSMProvider ssmProvider = ParamManager.getSsmProvider();
-    
-        // Retrieve a single parameter
-        String value = ssmProvider.get("/my/parameter");
+    ```java hl_lines="9 10"
+    import com.amazonaws.services.lambda.runtime.RequestHandler;
+    import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent;
```

Comment:
for all the snippets, can we simplify, by removing these APIGW events/responses, using a simple `RequestHandler<String, String>` and adding the handler method and end of the class `}`

Reply:
ACK